src/sql-parser/src/parser.rs

```diff
@@ -2124,11 +2124,33 @@ impl<'a> Parser<'a> {
                 })
             }
             LOKI => {
-                self.expect_keyword(ADDRESS)?;
-                let address = self.parse_literal_string()?;
+                let address = if self.parse_keyword(ADDRESS) {
+                    Some(self.parse_literal_string()?)
+                } else {
+                    None
+                };
```

Comment:
At what point does this error if neither the `LOKI_ADDR` env var is set nor the address is passed here? 🤔

Reply:
This is a good point. At connection time. We should validate this earlier. Ideally we would notify in the parser.